stdlib/2.7/inspect.pyi

```diff
@@ -8,41 +8,41 @@ ModuleInfo = NamedTuple('ModuleInfo', [('name', str),
                                        ('mode', str),
                                        ('module_type', int),
                                        ])
-def getmembers(object: object,
+def getmembers(object: Any,
                predicate: Callable[[Any], bool] = ...
-              ) -> List[Tuple[str, object]]: ...
+              ) -> List[Tuple[str, Any]]: ...
 def getmoduleinfo(path: str) -> Optional[ModuleInfo]: ...
 def getmodulename(path: str) -> Optional[str]: ...
 
-def ismodule(object: object) -> bool: ...
-def isclass(object: object) -> bool: ...
-def ismethod(object: object) -> bool: ...
-def isfunction(object: object) -> bool: ...
-def isisgeneratorfunction(object: object) -> bool: ...
-def isgenerator(object: object) -> bool: ...
-def istraceback(object: object) -> bool: ...
-def isframe(object: object) -> bool: ...
-def iscode(object: object) -> bool: ...
-def isbuiltin(object: object) -> bool: ...
-def isroutine(object: object) -> bool: ...
-def isabstract(object: object) -> bool: ...
-def ismethoddescriptor(object: object) -> bool: ...
-def isdatadescriptor(object: object) -> bool: ...
-def isgetsetdescriptor(object: object) -> bool: ...
-def ismemberdescriptor(object: object) -> bool: ...
+def ismodule(object: Any) -> bool: ...
```

Comment:
Honestly I would prefer not to have all that churn. For arguments in stubs, there is no difference between `object` and `Any`, both mean "any object". I actually (weakly) prefer using `object` here, but I don't think it's worth changing any code that has cargo-culted `Any` -- they're both right. But I definitely object to churn.